ecosystem/typescript/sdk/src/plugins/aptos_token.ts

```diff
@@ -549,4 +549,31 @@ export class AptosToken {
     );
     return txnHash;
   }
+
+  /**
+   * Burn an object by the object owner
+   * @param owner The object owner account
+   * @param objectId The object address
+   * @optional objectType. The object type, default to "0x1::object::ObjectCore"
+   * @returns The hash of the transaction submitted to the API
+   */
+  async burnObject(
+    owner: AptosAccount,
+    objectId: MaybeHexString,
+    objectType?: string,
+    extraArgs?: OptionalTransactionArgs,
+  ): Promise<string> {
+    const builder = new TransactionBuilderRemoteABI(this.provider, {
+      sender: owner.address(),
+      ...extraArgs,
+    });
+    const rawTxn = await builder.build(
+      "0x1::object::burn",
+      [objectType || "0x1::object::ObjectCore"],
+      [HexString.ensure(objectId).hex()],
+    );
```

Comment:
Noob q - If we don't provide this `burnObject()` function. I guess the 3rd party dev can just construct their own txn using this exact function id? `0x1::object::burn`?

Reply:
correct. nothing stops you or anyone else to simply build a transaction and submit it. These functions are helpers for developers to not worry about the transaction structure/build process and simply call a function with arguments.